backend/corpora/common/utils/dl_sources/url.py

```diff
@@ -82,10 +85,16 @@ def file_info(self) -> dict:
         resp = requests.head(self.url, allow_redirects=True)
         resp.raise_for_status()
 
-        print(resp.headers)
+        # Important: getting the content-length from Dropbox is unreliable, so we're setting a default value (100MB)
+        # to avoid breaking the upload pipeline.
+        try:
+            size = int(self._get_key_with_fallback(resp.headers, "content-length", "x-dropbox-content-length"))
```

Comment:
just checking that it is the processing of the response header, rather than the request itself, that is failing?

Reply:
The request itself works. It's a HEAD request to the download URL, so if it fails, the download won't work either, so I think it makes sense to fail fast here.